src/main/java/games/strategy/engine/lobby/server/db/HashedPassword.java

```diff
@@ -15,6 +19,14 @@ public HashedPassword(final String hashedPassword) {
    * Returns true if the hashed password looks like it could be a hash.
    */
   public boolean isValidSyntax() {
+    return isMd5Crypted() || isBcrypted();
+  }
+
+  public boolean isBcrypted() {
```

Comment:
How the password is hashed is a detail, there should not be a flag for knowing if this is bcrypted or otherwise. The check for encryption is just to check if the string looks valid. We can remove the `isValidSyntax` method TBH rather than upadte it, and we should really not have a public `isBcrypted` method.